source/04-PR/03.ptx

```diff
@@ -345,75 +230,78 @@
+                      <latex-image>
+                        \(
+                      \begin{array}{ r *{19}{c@{}} }
+                      &amp;  &amp; &amp; &amp; &amp; &amp; &amp;  \\ \cline{2-10}\\[-40.5pt]
+                      \multicolumn{1}{r}{\rlap{\kern4.5pt\smash{\raise.2ex\hbox{|}}}} &amp; &amp; &amp; &amp; &amp;?  &amp; &amp; ?  &amp; &amp; 7 \\
+                      &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp;\\ 
+
+
+                      \end{array}
+                        \)
+                      </latex-image>
+                      </image>
+                    </statement>
+                    </task>
+                    <task> 
+                      <statement>
+                        <p>Find the zero of the divisor. Put this in front of the other coefficients. </p>
+                        
+                          <image xml:id="example-synthetic-missing-divisor" width="20%">
+                            <latex-image>
+                              \(
+                                \begin{array}{ r *{19}{c@{}} }
+                                &amp;  &amp; &amp; &amp; &amp; &amp; &amp;  \\ \cline{2-10}\\[-40.5pt]
+                                \multicolumn{1}{r}{?\rlap{\kern4.5pt\smash{\raise.2ex\hbox{|}}}} &amp; &amp; &amp; &amp; &amp;?  &amp; &amp; ?  &amp; &amp; 7 \\
+                                &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp;\\ 
+          
+          
+                                \end{array}
+                              \)
+                            </latex-image>
+                            </image>
+                          </statement>
+                    </task>
+                    <task> 
+                      <statement>
+                        <p>How can we use the coefficients of the dividend and the zero of the divisor to produce the coefficients of the quotient in part b of Activity 4.3.8? </p>
+                        <image xml:id="example-synthetic-final_step" width="20%">
+                          <latex-image>
+                            \(
+                              \begin{array}{ r *{19}{c@{}} }
+                              &amp;  &amp; &amp; &amp; &amp; &amp; &amp;  \\ \cline{2-10}\\[-40.5pt]
+                              \multicolumn{1}{r}{?\rlap{\kern4.5pt\smash{\raise.2ex\hbox{|}}}} &amp; &amp; &amp; &amp; &amp; ?  &amp; &amp; ?  &amp; &amp; 7 \\
+                              &amp; &amp; &amp; &amp; &amp; &amp; &amp;? &amp; &amp; ? &amp; &amp;\\ 
+                              &amp; &amp; &amp; &amp; &amp; 3 &amp; &amp; 2 &amp; &amp;5 &amp; &amp; &amp; &amp;\\ 
+                              &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp; &amp;\\ 
+        
+        
+                              \end{array}
+                            \)
+                          </latex-image>
+                          </image>
+                        </statement>
+                    </task>
```

Comment:
```suggestion
                    <task> 
                      <statement>
                        <p>Using only the coefficients of the dividend, fill in the blanks in the top row below.</p>
                       
                      
                    <image xml:id="example-synthetic-division-coefficients" width="30%">
                      <latex-image>
                        \begin{tikzpicture}[x=1.1em, y=1.1em]
                          \draw[black!10] (-1,-1) rectangle (3,2);
                          \draw (0,2) -- (0,0) -- (3,0);
                          <!-- \node at (-0.5,1.5) {?}; -->
                          \node at (0.5,1.5) {?};
                          \node at (1.5,1.5) {?};
                          \node at (2.5,1.5) {7};
                          <!-- \node at (1.5,0.5) {?};
                          \node at (2.5,0.5) {?};
                          \node at (0.5,-0.5) {3};
                          \node at (1.5,-0.5) {2};
                          \node at (2.5,-0.5) {5}; -->
                        \end{tikzpicture}
                      </latex-image>
                      </image>
                    </statement>
                    </task>
                    <task> 
                      <statement>
                        <p>Find the zero of the divisor. Put this in front of the other coefficients. </p>
                        
                          <image xml:id="example-synthetic-missing-divisor" width="30%">
                            <latex-image>
                              \begin{tikzpicture}[x=1.1em, y=1.1em]
                                \draw[black!10] (-1,-1) rectangle (3,2);
                                \draw (0,2) -- (0,0) -- (3,0);
                                \node at (-0.5,1.5) {?};
                                \node at (0.5,1.5) {?};
                                \node at (1.5,1.5) {?};
                                \node at (2.5,1.5) {7};
                                <!-- \node at (1.5,0.5) {?};
                                \node at (2.5,0.5) {?};
                                \node at (0.5,-0.5) {3};
                                \node at (1.5,-0.5) {2};
                                \node at (2.5,-0.5) {5}; -->
                              \end{tikzpicture}
                            </latex-image>
                            </image>
                          </statement>
                    </task>
                    <task> 
                      <statement>
                        <p>How can we use the coefficients of the dividend and the zero of the divisor to produce the coefficients of the quotient in part b of Activity 4.3.8? </p>
                        <image xml:id="example-synthetic-final_step" width="30%">
                          <latex-image>
                            \begin{tikzpicture}[x=1.1em, y=1.1em]
                              \draw[black!10] (-1,-1) rectangle (3,2);
                              \draw (0,2) -- (0,0) -- (3,0);
                              \node at (-0.5,1.5) {?};
                              \node at (0.5,1.5) {?};
                              \node at (1.5,1.5) {?};
                              \node at (2.5,1.5) {7};
                              \node at (1.5,0.5) {?};
                              \node at (2.5,0.5) {?};
                              \node at (0.5,-0.5) {3};
                              \node at (1.5,-0.5) {2};
                              \node at (2.5,-0.5) {5};
                            \end{tikzpicture}
                          </latex-image>
                          </image>
                        </statement>
                    </task>
         <image xml:id="example-synthetic-division-coefficients" width="30%">
                      <latex-image>
                        \begin{tikzpicture}[x=1.1em, y=1.1em]
                          \draw[black!10] (-1,-1) rectangle (3,2);
                          \draw (0,2) -- (0,0) -- (3,0);
                          <!-- \node at (-0.5,1.5) {?}; -->
                          \node at (0.5,1.5) {?};
                          \node at (1.5,1.5) {?};
                          \node at (2.5,1.5) {7};
                          <!-- \node at (1.5,0.5) {?};
                          \node at (2.5,0.5) {?};
                          \node at (0.5,-0.5) {3};
                          \node at (1.5,-0.5) {2};
                          \node at (2.5,-0.5) {5}; -->
                        \end{tikzpicture}
                      </latex-image>
                      </image>
                    </statement>
                    </task>
                    <task> 
                      <statement>
                        <p>Find the zero of the divisor. Put this in front of the other coefficients. </p>
                        
                          <image xml:id="example-synthetic-missing-divisor" width="30%">
                            <latex-image>
                              \begin{tikzpicture}[x=1.1em, y=1.1em]
                                \draw[black!10] (-1,-1) rectangle (3,2);
                                \draw (0,2) -- (0,0) -- (3,0);
                                \node at (-0.5,1.5) {?};
                                \node at (0.5,1.5) {?};
                                \node at (1.5,1.5) {?};
                                \node at (2.5,1.5) {7};
                                <!-- \node at (1.5,0.5) {?};
                                \node at (2.5,0.5) {?};
                                \node at (0.5,-0.5) {3};
                                \node at (1.5,-0.5) {2};
                                \node at (2.5,-0.5) {5}; -->
                              \end{tikzpicture}
                            </latex-image>
                            </image>
                          </statement>
                    </task>
                    <task> 
                      <statement>
                        <p>How can we use the coefficients of the dividend and the zero of the divisor to produce the coefficients of the quotient in part b of Activity 4.3.8? </p>
                        <image xml:id="example-synthetic-final_step" width="30%">
                          <latex-image>
                            \begin{tikzpicture}[x=1.1em, y=1.1em]
                              \draw[black!10] (-1,-1) rectangle (3,2);
                              \draw (0,2) -- (0,0) -- (3,0);
                              \node at (-0.5,1.5) {?};
                              \node at (0.5,1.5) {?};
                              \node at (1.5,1.5) {?};
                              \node at (2.5,1.5) {7};
                              \node at (1.5,0.5) {?};
                              \node at (2.5,0.5) {?};
                              \node at (0.5,-0.5) {3};
                              \node at (1.5,-0.5) {2};
                              \node at (2.5,-0.5) {5};
                            \end{tikzpicture}
                          </latex-image>
                          </image>
                        </statement>
                    </task>
```

Reply:
Result:

![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/1559632/47adfe5b-055e-45bd-b0c7-593e72fce24b)

![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/1559632/91c33f0e-69b4-43e2-8b16-92abe1cab2d9)

Reply:
@sheppardw If you like @StevenClontz 's changes, there is a button right below his code that says "Commit suggestion" that you can hit to incorporate them.  And then when you are happy, mark this ready for review.